tox.ini

```
@@ -27,13 +28,14 @@ setenv =
     DJANGO_SETTINGS_MODULE = dbtemplates.test_settings
 deps =
     -r requirements/tests.txt
-    dj22: Django<2.3
-    dj32: Django<3.3
+    dj31: Django<3.2
```

```suggestion
    dj32: Django<3.3
```

And the envlist on lines 7 and 8 should be updated.